test/test.js

```diff
@@ -345,6 +345,15 @@ test('format values', (t) => {
   }
 })
 
+test('skip rows until', function (t) {
+  collect('junk_rows.csv', {skipLines: 3}, verify)
```

Comment:
If `skipLines` is `3` I would expect that the headers line would then be `ok,ok,ok!`. It looks like you need to account for the zero-based index there. To get the result the test is expecting, I would expect to pass `2` to the option.